0001_cars_test.cy.js

```diff
@@ -562,8 +561,11 @@ describe("Test Child:", () => {
       //act
       Common.RunSQL(cy, folderName, ["init_db_for_editing_a_child.sql"]);
       childVisit(childrenIndex);
+
       // cy.get(cyInterfaceCARS.page.socialWorker.page.children.view.child.tab.basicInfo).click();
       // cy.get(cyInterfaceCARS.page.socialWorker.page.children.view.child.page.basicInfo.tab.basicInfo).click();
+      childVisit(); // Reload Again
```

Comment:
we reload twice in a row? and it doesn't have `childrenIndex` from line 563

Reply:
I checked that I'm not sure why the data details of children on basic info tab don't show, this even when we click into the edit button, there was no data, so that's why I have to reload it again.